sentences/el/_common.yaml

```diff
@@ -1,12 +1,50 @@
 language: el
 responses:
   errors:
-    no_intent: TODO Sorry, I couldn't understand that
-    no_area: TODO No area named {{ area }}
-    no_domain: TODO {{ area }} does not contain a {{ domain }}
-    no_device_class: TODO {{ area }} does not contain a {{ device_class }}
-    no_entity: TODO No device or entity named {{ entity }}
-    handle_error: TODO An unexpected error occurred while handling the intent
-lists: {}
-expansion_rules: {}
-skip_words: []
+    no_intent: "Συγγνώμη, δεν μπόρεσα να το καταλάβω αυτό."
+    no_area: "Δεν υπάρχει περιοχή με όνομα {{ area }}"
+    no_domain: "Η περιοχή {{ area }} δεν περιέχει ένα {{ domain }}"
+    no_device_class: "Η περιοχή {{ area }} δεν περιέχει {{ device_class }}"
+    no_entity: "Δεν υπάρχει συσκευή ή οντότητα με όνομα {{ entity }}"
+    handle_error: "Προέκυψε ένα απροσδόκητο σφάλμα κατά το χειρισμό της πρόθεσης"
+lists:
+  color:
+    values:
+      - "λευκό"
+      - "μαύρο"
+      - "κόκκινο"
+      - "πορτοκαλί"
+      - "κίτρινο"
+      - "πράσινο"
+      - "μπλε"
+      - "μωβ"
+      - "καφέ"
+  brightness:
+    range:
+      type: "ποσοστό"
+      from: 0
+      to: 100
+  temperature:
+    range:
+      type: "θερμοκρασία"
+      from: 0
+      to: 100
+ temperature_unit:
+    values:
+      - "celsius"
+      - in: "Κελσίου"
+        out: "celsius"
+      - "fahrenheit"
+      - in: "Φαρενάιτ"
+        out: "fahrenheit"
+expansion_rules:
+  name: "[(ο | τον | η | την | το )] {name}"
+  area: "[(ο | τον | η | την | το )] {area}"
+  what_is: "(τι έιναι | ποιος έιναι | ποια είναι | ποιο είναι)"
+  brightness: "{brightness:brightness_pct} [Ποσοστό]"
+  turn: "(άλλαξε | γύρισε | γύρνα)"
+  temp: "θερμοκρασία"
+  temperature: "{temperature} [βαθμοί] [{temperature_unit}]"
```

Comment:
The keys (`what_is`) should be translated. And they should also be used in sentences. You shouldn't add ones just because they exist in English. Greek probably works differently.